### PR TITLE
Fixing param order

### PR DESCRIPTION
**Pull request by josenavas**
Last PR, this fixes the last bit of the plugin so it works with Qiita.

### CODE REVIEW

File: qp_deblur/deblur.py [hunk 1]
```diff
@@ -142,7 +142,7 @@ def deblur(qclient, job_id, parameters, out_dir):
         # a valid BIOM, so we are going to create an empty table
         t = Table([], [], [])
         with biom_open(final_biom_16s, 'w') as f:
-            t.to_hdf5('qp-deblur generated', f)
+            t.to_hdf5(f, 'qp-deblur generated')
```
Comment (mortonjt) [inline qp_deblur/deblur.py]: crap - how did we let that slip through the cracks XD

**Comment by mortonjt** [comment]
This seems like just a silly mistake - do you think it warrants a unittest?

If not - I think it is good to go.

**Comment by josenavas** [comment]
The problem that I have with this is that I don't know how to generate a good test case that triggers this error. I'm testing this in the test environment using a specific dataset that I think would be to big to add to the repo for testing.

**Comment by mortonjt** [comment]
Ok.  For this given case I wouldn't worry about it given that this is pretty easy to catch.  I just wonder how we missed this ...

**Comment by mortonjt** [comment]
@antgonza want to merge this?

**Comment by antgonza** [comment]
@mortonjt, nothing is perfect and sometime this happens, even after N reviews. Anyway, merging.

**Pull request merged**